apps/server/src/common/filters/mongo-server-error.filter.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  ExceptionFilter,
+  Catch,
+  ArgumentsHost,
+  HttpStatus,
+} from '@nestjs/common';
+import { Response } from 'express';
+import { mongo } from 'mongoose';
+
+@Catch(mongo.MongoServerError)
+export class MongoErrorFilter implements ExceptionFilter {
```

Comment:
Could you rename this class to something like 'DuplicateKeyErrorFilter' to be more related to what it does?